ssl/record/methods/tls_common.c

```diff
@@ -1916,8 +1916,11 @@ int tls_retry_write_records(OSSL_RECORD_LAYER *rl)
             } else {
                 if (BIO_should_retry(rl->bio))
                     ret = OSSL_RECORD_RETURN_RETRY;
-                else
+                else {
```

Comment:
OpenSSL style says that if one branch of an if uses `{}` then both branches should do. So please can you insert the missing `{}` above?

Reply:
fixed, and dropped branches other than master